doc/source/dev/toolchain.rst

```diff
@@ -90,8 +91,7 @@ Compilers
 ^^^^^^^^^
 
 Building SciPy requires compilers for C, C++, Fortran, as well as the
-python transpilers Cython and Pythran (the latter is an opt-out dependency
-as of version 1.7.0).
+python transpilers Cython and Pythran.
```

Comment:
Did you remove the optional deactivation of Pythran in anticipation of its eventual obligate activation? I still see `SCIPY_USE_PYTHRAN` as a switch in various places.

Reply:
I had made this change a while ago, and I think the intention was to leave the details for the `Cython & Pythran` section (where it's still noted as opt-out). I'm not proposing to make pythran mandatory here, and happy to keep the formulation if that's desired.

Reply:
I reinstated that sentence.